.github/workflows/bitcoin-tests.yml

```diff
@@ -1,19 +0,0 @@
-name: Run bitcoind/stacks-node integration tests
-
-on: pull_request
```

Comment:
This workflow would be good to run on all PRs, I think by removing this workflow and merging it into `stacks-blockchain.yml`, it would only run on `v*` tag pushes.

Reply:
This workflow will now run on all PRs